### ISSUE TITLE
The text written in the search bar disappears in AddClockActivity after a screen orientation change

### ISSUE THREAD

**Issue by dlicse** [dasbiswajit/worldclockwidget#11]
When I change the orientation of the device screen, the text written in the search bar disappears
![2019-07-22 10-23-33_before](https://user-images.githubusercontent.com/53469180/62287220-b4ce0380-b459-11e9-915e-c17b0c025e81.png)
![2019-07-22 10-23-33_after](https://user-images.githubusercontent.com/53469180/62287222-b5669a00-b459-11e9-9259-e415e9b3fe6c.png)

**Comment by dasbiswajit** [comment]
Can you try to click the magnify glass again.

**Comment by dlicse** [comment]
Yes, I can click the magnifying glass again, but the search bar is empty and the value previously entered is lost.

We would like to know if you consider it as exepcted behaviour or as a GUI failure. 

Thank you in advance!
